src/pages/core/plugins/update-delegate.md

```diff
@@ -19,7 +20,13 @@ The Update Delegate Plugin can be used when:
 
 ## Arguments
 
-The Update Delegate Plugin doesn't contain any arguments to pass in.
+|                     |             |
+| ------------------- | ----------- |
+| additionalDelegates | publickey[] |
+
+### additionalDelegates
+
+Additional delegates allow you to add more than one delegate to the updateDelegate plugin. These additional delegates do come with some restrictions such as you can't update the Collection as an Additioanl Delegate address.
```

Comment:
There's some other ones I think we should list as well, listed in the PR description